synedrion/src/cggmp21/params.rs

```diff
@@ -9,8 +9,9 @@ use crate::uint::{
 };
 
 use serde::{Deserialize, Serialize};
+use zeroize::Zeroize;
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize, Zeroize)]
```

Comment:
Strange that it has to be `Zeroize`, I guess the requirement of an overly needy heuristic in a derivation macro?

Reply:
It doesn't, good catch. I thought `PaillierParams` needed to be `Zeroize` though? Like, if I remove
the requirement
[over here](https://github.com/dvdplm/synedrion/blob/dcb95ce721d95c903c4c459b9a7806ffd67138af/synedrion/src/paillier/params.rs#L11)
then it is no longer needed here of course. On the other hand, this means that we need to be careful
to ensure all components of `PaillierParams` that may contain secrets are zeroized properly. I
believe this is the case at the moment.